### PR TITLE
Show mailbox counter if sub mailbox has unread messages

### PR DESCRIPTION
**Pull request by ChristophWurst**
Fixes https://github.com/nextcloud/mail/issues/6628

Bildschirmfoto vom 2022-08-08 23-06-52

### CODE REVIEW

**Review by st3iny — COMMENTED** [review]
Tested and works. However, the design could be improved.

How it looks with a 2-digit notification counter:
image

**Comment by ChristophWurst** [comment]
> Tested and works. However, the design could be improved.
> 
> How it looks with a 2-digit notification counter:

Hmm. Maybe a non-breakable whitespace?

**Comment by ChristophWurst** [comment]
Bildschirmfoto vom 2022-08-09 15-23-07

nbsp looks acceptable

File: src/components/NavigationMailbox.vue [hunk 1]
```diff
@@ -174,7 +174,10 @@
 				{{ t('mail', 'Delete mailbox') }}
 			</ActionButton>
 		</template>
-		<AppNavigationCounter v-if="showUnreadCounter && mailbox.specialRole !== 'trash'" slot="counter">
+		<AppNavigationCounter v-if="showUnreadCounter && subCounter" slot="counter">
```
Comment (miaulalala) [inline src/components/NavigationMailbox.vue:177]: wouldn't this only show the unread counter if there was an unread item in the submailbox?

**Review by JuliaKirschenheuter — CHANGES_REQUESTED** [review]
Works! Thank you!

I would remove ```max-width: 44px;```
because of 
image
there are a lot of space

File: src/components/NavigationMailbox.vue (hunk 1 above)
Comment (ChristophWurst) [reply src/components/NavigationMailbox.vue:177]: this is for unread messages in a child. the one for the parent is in the else-if

**Review by JuliaKirschenheuter — APPROVED** [review]
works!

**Review by GretaD — APPROVED** [review]
it works and looks nice

**Pull request merged**